grpc-sys/Cargo.toml

```diff
@@ -31,12 +31,6 @@ exclude = [
     "grpc/third_party/abseil-cpp/*",
     "grpc/third_party/benchmark/*",
     "grpc/third_party/bloaty/*",
-    "grpc/third_party/boringssl/fuzz/*",
-    "grpc/third_party/boringssl/crypto/cipher_extra/test/*",
-    "grpc/third_party/boringssl/third_party/android-cmake/*",
-    "grpc/third_party/boringssl/third_party/wycheproof_testvectors",
-    "grpc/third_party/boringssl/third_party/googletest",
-    "!grpc/third_party/boringssl/third_party/googletest/src/gtest-all.cc",
     "grpc/third_party/boringssl-with-bazel/*",
     "grpc/third_party/gflags/*",
     "grpc/third_party/googletest/*",
```

Comment:
You better check if these dir are still needed now, I don't know if some of them are necessary @BusyJay

Reply:
For example, `gflags` and `googletest` is not necessary to compile grpc-rs